internal/layers/layers.go

```diff
@@ -71,16 +77,16 @@ func (layers *ImageLayers) Release(ctx context.Context, all bool) error {
 // Returns the path at which the `rootfs` of the container can be accessed. Also, returns the path inside the
 // UVM at which container scratch directory is located. Usually, this path is the path at which the container
 // scratch VHD is mounted. However, in case of scratch sharing this is a directory under the UVM scratch.
-func MountLCOWLayers(ctx context.Context, containerID string, layerFolders []string, guestRoot, volumeMountPath string, vm *uvm.UtilityVM) (_, _ string, err error) {
+func MountLCOWLayers(ctx context.Context, containerID string, layerFolders []string, guestRoot, volumeMountPath string, vm *uvm.UtilityVM) (_, _ string, _ []*LayerMount, err error) {
 	if vm.OS() != "linux" {
-		return "", "", errors.New("MountLCOWLayers should only be called for LCOW")
+		return "", "", nil, errors.New("MountLCOWLayers should only be called for LCOW")
 	}
 
 	// V2 UVM
 	log.G(ctx).WithField("os", vm.OS()).Debug("hcsshim::MountLCOWLayers V2 UVM")
 
 	var (
-		layersAdded       []string
+		layersAdded       []*LayerMount
 		lcowUvmLayerPaths []string
```

Comment:
Do we really need both of these? It seems like `lcowUvmLayerPaths` is all information we can get from `layersAdded`.

Reply:
I kept the `lcowUvmLayerPaths` here because we needed just the uvm paths later in the function. So rather than loop through all the `layersAdded` to get that later, I just stored the uvm paths here like we were already doing.

Reply:
I see. Can we add a comment here explaining why they are separate then?